src/main/js/CollapsibleContainer.tsx

```diff
@@ -66,17 +77,27 @@ const CollapsibleContainer: FC<Props> = ({ title, separatedEntries, emptyMessage
     if (!childArray || childArray.length === 0) {
       content = <EmptyMessage messageKey={emptyMessage} />;
     } else if (separatedEntries) {
-      content = childArray.map(child => <Content>{child}</Content>);
+      content = childArray.map(child => <Content className="p-2">{child}</Content>);
     } else {
-      content = <Content className="box">{children}</Content>;
+      if (contentWrapper) {
+        content = React.createElement(contentWrapper, null, children);
+      } else {
+        content = <Content className="box p-2">{children}</Content>;
+      }
     }
   }
 
+  const onClick = () => {
+    const newState = !collapsed;
+    setCollapsed(newState);
+    onCollapseToggle && onCollapseToggle();
+  };
+
   return (
     <Container>
-      <div className="has-cursor-pointer" onClick={() => setCollapsed(!collapsed)}>
+      <div className="has-cursor-pointer" onClick={onClick}>
         <Headline>
-          <Icon name={icon} color="default" /> {title}
+          <Icon name={icon} color="default" /> {title} <Tag color="info">{count || 0}</Tag>
```

Comment:
I'm not satisfied with the tag yet. We should make the font bold and maybe a little bigger?